app/controllers/api/account.php

```diff
@@ -1500,25 +1500,27 @@
         $response->dynamic($user, Response::MODEL_USER);
     });
 
-App::delete('/v1/account')
-    ->desc('Delete Account')
+App::patch('/v1/account')
+    ->desc('Update Status (Block)')
```

Comment:
```suggestion
    ->desc('Update Account Status')
```

Let's be consistent with the users service